client/my-sites/plugins/plugin-details-notices.jsx

```diff
@@ -0,0 +1,59 @@
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import Notice from 'calypso/components/notice';
+import { getPluginPurchased } from 'calypso/lib/plugins/utils';
+import {
+	AUTOMOMANAGED_PLUGINS,
+	BUNDLED_PLUGINS_BY_PLAN,
+	PREINSTALLED_PLUGINS,
+} from 'calypso/my-sites/plugins/constants';
+import {
+	getSitePurchases,
+	hasLoadedSitePurchasesFromServer,
+} from 'calypso/state/purchases/selectors';
+
+const PluginDetailsNotices = ( { selectedSite, plugin, translate } ) => {
+	const hasLoadedSitePurchases = useSelector( hasLoadedSitePurchasesFromServer );
+	const isFullPluginAndPurchasesFetched = hasLoadedSitePurchases && plugin?.fetched;
+	const isWpcomPreinstalled =
+		PREINSTALLED_PLUGINS.includes( plugin.slug ) || AUTOMOMANAGED_PLUGINS.includes( plugin.slug );
+	const isBundledPlugin = BUNDLED_PLUGINS_BY_PLAN[ selectedSite?.plan?.product_slug ]?.includes(
```

Comment:
`ecommerce-bundle` is the product slug of `ecommerce` annual plan. The monthly one is `ecommerce-bundle-monthly`. We also have `ecommerce-bundle-2y` and might have more variations in the future. Seems like `isSiteOnEcommerce` would be a better variable to check against. https://github.com/Automattic/wp-calypso/blob/4193801455167437405e54230134c45750a7b779/client/state/sites/plans/selectors/is-site-on-ecommerce.ts#L17

![CleanShot 2023-05-04 at 16 03 05@2x](https://user-images.githubusercontent.com/12430020/236213059-a10cd55a-9efb-4fe5-a684-c9625227a455.jpg)

Reply:
I fixed the issue by using the method you suggested. Thank you, @cpapazoglou !